coalib/bears/requirements/PackageRequirement.py

```diff
@@ -0,0 +1,43 @@
+from coala_decorators.decorators import generate_eq
+from coala_decorators.decorators import generate_repr
```

Comment:
hm wait, this statement does not exceed the 80 char limit at all!
this is a bug in coala, ignore that one^^